### PR TITLE
terraform example - mysql and datadog integration

### PR DESCRIPTION
**Pull request by fatimafirdous**
# Included a terraform example for MySQL integration with datadog

### CODE REVIEW

File: terraform/mysql/mysql-datadog.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+resource "aiven_service_integration_endpoint" "datadog" {
+  project       = var.project_name
+  endpoint_name = "my-Datadog"
+  endpoint_type = "datadog"
+
+  datadog_user_config {
+    datadog_api_key = var.datadog_api_key
+    datadog_tags {
+      tag = "environment:dev"
```
Comment (paulgrainger-aiven) [inline terraform/mysql/mysql-datadog.tf:16]: imo this should be a variable too, rather than hardcoded

File: terraform/mysql/provider.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    aiven = {
+      source = "aiven/aiven"
+    version = ">= 2.0.0, < 3.0.0" }
```
Comment (paulgrainger-aiven) [inline terraform/mysql/provider.tf:5]: the latest version of the Aiven terraform provider is 3.10.0. I see this comes from the docs though, so perhaps they are out of date

**Review by paulgrainger-aiven — DISMISSED** [review]

File: terraform/mysql/provider.tf (hunk 2 above)
Comment (fatimafirdous) [reply terraform/mysql/provider.tf]: Made the changes

File: terraform/mysql/mysql-datadog.tf (hunk 1 above)
Comment (fatimafirdous) [reply terraform/mysql/mysql-datadog.tf]: Made the required changes

**Review by paulgrainger-aiven — APPROVED** [review]

**Review by mparikhaiven — APPROVED** [review]
LGTM

**Pull request merged**